python/py-spdx-lookup/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-spdx-lookup
+version             0.3.2
+revision            0
+
+maintainers         nomaintainer
+description         SPDX license list query tool
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://github.com/bbqsrc/spdx-lookup-python
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+license             2-clause BSD License
+checksums           sha256  d81e16ace58f668be3c2a8502355269e1661083a0c8538015945821469f6a5fa \
+                    rmd160  3ba50ad1b5d273d5c8e426d4d8375a3e2b6e5a56 \
+                    size    4763
+                                
+python.versions     37
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                        port:py${python.version}-setuptools
```

Comment:
in this case ```py-setuptools``` is not only a build dependency as the package install a ```console_scrip``` and, therefore, needs ```setuptools``` at runtime as well. So here you should add it instead under ```depends_lib```.  I am not sure if this can be deduced automatically...

(As a side note: the distinction between ```depends_lib``` and ```depends_run``` isn't used a lot in the Python ports I have looked at and I am not sure if it really matters. But, we should certainly investigate this a bit more and implement it correctly within the ```upt``` backend.)

Reply:
Stupid nitpicking (aka: please ignore): if you are already putting dependencies to the next line, you can align to 20 spaces.